pyro/distributions/__init__.py

```diff
@@ -28,6 +30,7 @@
 from pyro.distributions.transforms import *  # noqa F403
 
 __all__ = [
+    "constraints",
```

Comment:
you can remove this line, it was updated in #2052

Reply:
done.